lighthouse-core/gather/gather-runner.js

```diff
@@ -91,13 +91,14 @@ class GatherRunner {
 
   static setupDriver(driver, options) {
     log.log('status', 'Initializing…');
+    const resetStorage = !(options.flags && options.flags.disableStorageReset);
```

Comment:
oh yeah it probably always is, just was jumping around to other places where that wasn't true. i'll update